mbed_flasher/erase.py

```diff
@@ -95,20 +112,26 @@ def erase_board(self, mount_point, serial_port):
             self.logger.info("erasing device")
             with open(join(mount_point, 'ERASE.ACT'), 'wb') as new_file:
                 pass
-            sleep(SLEEP_TIME_FOR_REMOUNT)
-            success = self.reset_board(serial_port)
-            if isfile(join(mount_point, 'ERASE.ACT')):
-                success = EXIT_CODE_ERASE_FAILED_NOT_SUPPORTED
-            if success != 0:
-                self.logger.error("erase failed")
-                return success
+            t = Thread(target=self.runner, args=([mount_point, 'ERASE.ACT'],))
+            t.start()
+            if not no_reset:
```

Comment:
So, the board will be reset even before we finished waiting for remount? Is this right?

After remount the MCU will be left in some kind of jammed state. I think it should be safe to reset it afterwards